docs/semgrep-code/editor.md

```diff
@@ -244,6 +246,10 @@ To add a rule to the **Policies** page:
 
 If successful, you'll see a pop-up window indicating that your rule has been added.
 
+## Organize private rules
+
+All private rules for an organization are saved to the organization's folder. To further organize rules, consider organizational naming conventions to keep related rules near each other, such as by internal team, rule language, or vulnerability category.
```

Comment:
I regret not thinking of this response when I got a ticket about it today

Reply:
actually, wait ... can you sort by name in the editor? I don't see a way to do that, and without it, namespacing doesn't help

Reply:
oh...good call. let me check

Reply:
yeah no, I was making that up. Darn. Maybe we can put in a request to make it work that way...

Reply:
I think I'll revise it to "to make it easier to find/search for rules", how does that sound?

Reply:
I think that's fine, although including this section at all still feels a little like advertising what the product can't do yet. it hasn't come up enough for me to have strong feelings though.

Reply:
I have no strong feelings about keeping or removing, but I do think giving the user a tip to follow a naming convention (as we have nothing else) helps them use the feature; an organizational tip is better than nothing :)